docs/user/aws/customization.md

```diff
@@ -1,18 +1,50 @@
 # AWS Platform Customization
 
-The following options are available when using AWS:
+Beyond the [platform-agnostic `install-config.yaml` properties](../customization.md#platform-customization), the installer supports additional, AWS-specific properties.
 
-- `machines.platform.aws.rootVolume.iops` - the reserved IOPS of the root volume
-- `machines.platform.aws.rootVolume.size` - the size (in GiB) of the root volume
-- `machines.platform.aws.rootVolume.type` - the storage type of the root volume
-- `machines.platform.aws.type` - the EC2 instance type
-- `machines.platform.aws.zones` - a list of the availability zones that the installer will use when creating machines of this pool
-- `platform.aws.region` - the AWS region that the installer will use when creating resources
-- `platform.aws.userTags` - a map of keys and values that the installer will add as tags to all resources it creates
+## Cluster-scoped properties
+
+* `amiID` (optional string): The AMI that should be used to boot machines for the cluster.
```

Comment:
should string be surrounded with back ticks?

Reply:
> should string be surrounded with back ticks?

Like:

```markdown
* `amiID` (optional `string`): ...
```

?  I think it's fine without that; if we wanted to be formal we could link to the [JSON RFC for string][1] or [the YAML string spec][2].  But the YAML spec is pretty dense; I'm fine just leaving it as an informal "string" and expecting folks to figure it out.  I'm also fine linking to the specs if you want me to file that as follow-up work.

[1]: https://tools.ietf.org/html/rfc8259#section-7
[2]: https://yaml.org/spec/1.2/spec.html#id2802842